src/pkgmt/versioneer.py

```diff
@@ -107,17 +113,24 @@ def version(
     6. Commit new development version and push
 
     Create a new version (projects without setup.py) :
-    1. These projects should contain two essential files:
-       config.yaml in root directory which should contain the repo
-       name
-       _version.py file containing __version__ in the required directory
-    2. version_package will be the directory containing _version.py file
-    3. Set new stable version in package_name/_version.py
+    1. These projects should contain `version_file` entry in pyproject.toml file.
+       version_file should be the path to the version file.
+    3. Set new stable version in `version_file`.
```

Comment:
let's move all of these instructions to the documentation. the docstring here should only contain a brief description of what this function does